ws/tracing/tracing.go

```diff
@@ -0,0 +1,64 @@
+package tracing
+
+import (
+	"context"
+
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/mainflux/mainflux/ws"
+	"github.com/opentracing/opentracing-go"
+)
+
+var _ ws.Service = (*tracingMiddleware)(nil)
+
+const (
+	publish_op     = "publish_op"
+	subscribe_op   = "subscribe_op"
+	unsubscribe_op = "unsubscribe_op"
+)
+
+type tracingMiddleware struct {
+	tracer opentracing.Tracer
+	svc    ws.Service
+}
+
+// New creates new ws tracing middleware service
+func New(tracer opentracing.Tracer, svc ws.Service) ws.Service {
+	return &tracingMiddleware{
+		tracer: tracer,
+		svc:    svc,
+	}
+}
+
+// Publish implements ws.Service
```

Comment:
Update generic comments. The same goes for all the other exported Service methods.